Created root to signup and login page

Albertwo512 opened the pull request:
I create diferents roots for each page o component, and protect page that the user login only can watch, if you're not logging, you can't watch that page.
![Test1](https://github.com/Albertwo512/X-B2/assets/122117432/198595f3-9f44-4e52-9aa6-51a720d13848)

Albertwo512 committed:
Created root to signup and login page

I create diferents roots for each page o component, and protect page that the user login only can watch, if you're not logging, you can't watch that page.

membriux commented on x-b2/src/Auth.js:
```diff
@@ -0,0 +1,106 @@
+    });
+  };
+
+  const signout = (cb) => {
```
Please implement the `Parse.User.Logout()` before the `setUser(null)`. Here is an example:
https://www.back4app.com/docs/react-native/parse-sdk/working-with-users/react-native-login#BpiHW
![Screenshot 2024-05-31 at 1 04 47 PM](https://github.com/Albertwo512/X-B2/assets/20372706/33eb6a38-d880-4c94-8608-5fb9a3102b25)

membriux commented on x-b2/src/example.js:
```diff
@@ -0,0 +1,184 @@
+import React, { useContext, createContext, useState } from "react";
```
Lets remove this file since we don't need it anymore

membriux reviewed (changes requested):
Please go over the comments I left

Albertwo512 committed:
changes complete

i delete the component "example.js" because i don’t need, also the logout component, and the compoment that you told me, it's of react-native.
